### PR TITLE
fix(android): Improve gen 1 TOTP docs

### PR DESCRIPTION
**Pull request by mattcreaser**
#### Description of changes:

- Add missing docs for Android for MFA selection and TOTP setup in Multi-Step Sign In page.
- Fix incorrect docs for MFA selection in Manage MFA Settings (was incorrectly showing confirm with TOTP content)
- Convert Fragment to InlineFilter on affected sections.
 
#### Related GitHub issue #, if available:
#6874 

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [x] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] Swift
- [x] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [x] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [x] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [x] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [x] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://docs.amplify.aws/)` 
            HTML: `<a href="https://docs.amplify.aws/">link</a>`_

### When this PR is ready to merge, please check the box below
- [x] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

**Comment by mattcreaser** [comment]
No redirects are needed because I replaced the deleted fragment with an InlineFilter.

File: src/fragments/lib/auth/native_common/signin_next_steps/common.mdx [hunk 1]
```diff
@@ -43,6 +43,96 @@ import iosConfirmSignInTotpMFA from '/src/fragments/lib/auth/ios/signin_next_ste
...
+)
+```
+</Block>
+
+<Block name="Kotlin - Coroutines">
+
+```kotlin
+try {
+    val result = Amplify.Auth.confirmSignIn(
+        "confirmation code"
```
Comment (tjleing) [inline src/fragments/lib/auth/native_common/signin_next_steps/common.mdx:97]: nit:
```suggestion
        "TOTP confirmation code"
```
to match

**Review by tjleing — APPROVED** [review]
approving with a nit, looks good!

File: src/fragments/lib/auth/native_common/signin_next_steps/common.mdx (hunk 1 above)
Comment (mattcreaser) [reply src/fragments/lib/auth/native_common/signin_next_steps/common.mdx]: Thanks good catch - had copied this from the existing fragment 😄

**Review by josefaidt — APPROVED** [review]

**Review by dbanksdesign — APPROVED** [review]

**Review by vincetran — APPROVED** [review]

**Pull request merged**